### DISCUSSION TITLE
Improvements to public transport

### DISCUSSION THREAD

**Discussion by TristanGah** [voxeltycoon/issues#367] [General]
Hi,
so I am certain some of these points are already discussed and will be revised, but I have a few additions.
Currently it feels like all the working hours went into designing the "factorio aspect" of the game, and that is really nice. But the flow of people is really boring and a pain to build.
I think bus stations shouldn't demolish a house, that just doesn't make sense and is incredibly expensive early on. Also we should be able to see which parts are already covered. 
But what I really wish for, is that people and factories get combined. Maybe make it so that we need to deliver people to factories as workers and that they work less efficient if not enough workers are there. this would also work well with the day/night cycle where more people want to travel at certain times. Obviously the whole transport system would need to make more sense, that people have real destinations and can switch vehicles etc.

**Comment by katepolak** [comment]
It is really that more time went into other portions of the game, the main reason for public transport being incomplete is because it is brand new in this version and is far from complete, the game is in early access after all. So expect changes and improvement to the whole passenger system. I will keep this suggestion open though, as you do have some interesting ideas in the latter part of the suggestion.

**Comment by TristanGah** [comment]
Hi, thanks for replying. Sorry if my first comment sounded offensively.
